charts/tikv-importer/templates/_helpers.tpl

```diff
@@ -0,0 +1,25 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "chart.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "cluster.name" -}}
+{{- default .Release.Name .Values.clusterName }}
```

Comment:
`.Release.Name` here will not be the target TiDB Cluster name because release names cannot be duplicated, just use the `.Values.clusterName`.